Src/libCZI/CMakeLists.txt

```diff
@@ -246,17 +246,21 @@ else()
     GIT_TAG "v1.5.6"
   )
 
-  if(NOT zstd_POPULATED)
-    message(STATUS "Fetching zstd...")
-    set(ZSTD_BUILD_PROGRAMS  OFF CACHE BOOL "" FORCE)
-    set(ZSTD_BUILD_SHARED    OFF CACHE BOOL "" FORCE)
-    set(ZSTD_BUILD_TESTS     OFF CACHE BOOL "" FORCE)
-    set(ZSTD_BUILD_CONTRIB   OFF CACHE BOOL "" FORCE)
+  message(STATUS "Fetching zstd...")
+  set(ZSTD_BUILD_PROGRAMS  OFF CACHE BOOL "" FORCE)
+  set(ZSTD_BUILD_SHARED    OFF CACHE BOOL "" FORCE)
+  set(ZSTD_BUILD_TESTS     OFF CACHE BOOL "" FORCE)
+  set(ZSTD_BUILD_CONTRIB   OFF CACHE BOOL "" FORCE)
 
-    FetchContent_Populate(zstd)
+  FetchContent_MakeAvailable(zstd)
 
-    add_subdirectory(${zstd_SOURCE_DIR}/build/cmake ${zstd_BINARY_DIR})
-  endif()
+  add_subdirectory(${zstd_SOURCE_DIR}/build/cmake ${zstd_BINARY_DIR})
+endif()
+
+if (LIBCZI_HAVE_WIN32_API)
+  set(libCZI_WINDOWSAPIAVAILABLE 1)
+else()
```

Comment:
Simply not setting it otherwise has similira effect right? You want to be explicit here, I reckon?

Reply:
Not setting libCZI_WINDOWSAPIAVAILABLE would have the effect that in the resulting config-header-file the variable would be replaced with "" (empty string) - which would probably not work in current version. So, no - in current version it has to read either "0" or "1" and nothing else.